### PR TITLE
Added the ability to select an alternative editor from the UD

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Added the ability to select an alternative editor from the dashboard.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/20627

Знімок екрана 2022-07-13 о 13 57 18

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]
great job, works like a charm against dogfooding :+1: 
Let's wait for UX approval before merging

**Comment by beaumorley** [comment]
Looks great!

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**
